### DISCUSSION TITLE
How do you use the HrefLangs component?

### DISCUSSION THREAD

**Discussion by m1thrandir225** [Alexandre-Fernandez/astro-i18n#31] [Q&A]
Hello as the title says, how do you actually use the HrefLangs component and do you need it? There is one line in the documentation and I can't import it from the astro-i18n package.

Thanks

**Comment by Alexandre-Fernandez** [comment]
```jsx
---
import { HrefLangs } from "astro-i18n/components"
---

<head>
	<HrefLangs />
</head>

```

EDIT: 
By the way this component assumes there's a translation of your page for every language. It will generate a "hreflang" for every lang in the config even if it doesn't exists in your routes. Most of the time this is not a problem but in some edge cases it might be.
You can check the [source code](https://github.com/Alexandre-Fernandez/astro-i18n/blob/main/src/astro/HrefLangs.astro), it's just a regular Astro component.

**Comment by abdelrazzaq-dev** [comment]
Can we add this to the docs?

**Reply by Alexandre-Fernandez** [reply]
You're free to make a pull request
